tests/software_tests/can/addressing/test_abstract_addressing_information.py

```
@@ -0,0 +1,335 @@
+import pytest
+from mock import Mock, patch
```

Ruff (0.12.2)

`mock` is deprecated, use `unittest.mock`

Import from `unittest.mock` instead

(UP026)